app/liquid_tags/jsitor_tag.rb

```diff
@@ -21,18 +22,21 @@ def render(_context)
 
   def jsitor_link_parser(link)
     parsed_link = ActionController::Base.helpers.strip_tags(link.strip).gsub("amp;", "")
-
-    return parsed_link if link_valid?(parsed_link)
-
-    jsitor_error
+    link_valid?(parsed_link)
   end
 
   def link_valid?(link)
```

Comment:
this should probably be called `validate_link` since it raises an exception.

Otherwise you can have `jsitor_link_parser` do `link_valid?() || jsitor_error`

Reply:
ill just change the method name. :smile: `validate_link`